README.rst

```diff
@@ -15,18 +15,29 @@ Building
 
 To build the docs locally, first install ``sphinx``, ``furo`` and
 ``sphinx-copybutton``.
-On Fedora this can be with::
+
+On **Fedora** this can be done with::
 
   sudo dnf install python3-sphinx python3-furo python-sphinx-copybutton python-sphinxext-opengraph
 
-On Debian this can be with::
+On **Debian** this can be done with::
 
   sudo apt install python3-sphinx python3-furo python-sphinx-copybutton python-sphinxext-opengraph
 
-Then run ``make html`` in the ``docs`` directory.
+With **pip**, this can be done with::
+
+  pip install -r requirements.txt
+
+Then, build the documentation::
+
+  cd docs
+  make html
+
+Then, you can run an HTTP server and follow the printed link
+(`localhost:8000 <http://localhost:8000>`_)
```

Comment:
I think the port might actually be different, if 8000 is already taken

Reply:
It is 8000 and does not change:

    OSError: [Errno 98] Address already in use

Reply:
8000 is the default, if the port is used by something else locally it will fail.  You can specify a custom port with 

python3 -m http.server <PORT>

You can document it if you want

Reply:
It is a rare case and not worth the space as most people will not run into this problem.